### PR TITLE
[kommander] docs: Add line to review app requirements on install/config pages

### PR DESCRIPTION
**Pull request by gracedo**
## Description of changes being made
Add some notes to the kommander install docs to review the mgmt cluster and workspace app requirements to ensure cluster has sufficient resources prior to installing kommander.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4400.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/install/networked/index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Before you begin using Kommander, you must:
 
 - Configure a Konvoy cluster using the [Konvoy Install](/dkp/konvoy/2.1/choose-infrastructure/) instructions.
 - Install `cert-manager` prior to installing Kommander. (Konvoy does this as part of its install process).
+- Review [Management cluster application requirements](../mgmt-cluster-apps) and [Workspace platform application requirements](../../workspaces/applications/platform-applications/platform-application-requirements) to ensure that your cluster has sufficient resources.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/networked/index.md:16]:
```suggestion
- Review the [Management cluster application requirements](../mgmt-cluster-apps) and [Workspace platform application requirements](../../workspaces/applications/platform-applications/platform-application-requirements) to ensure that your cluster has sufficient resources.
```

File: pages/dkp/kommander/2.2/install/configuration/index.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ enterprise: false
 
 You can configure Kommander during the initial installation, and also post-installation using the Kommander CLI.
 
+<p class="message--note"><strong>NOTE:</strong> Review <a href="../mgmt-cluster-apps">Management cluster application requirements</a> and <a href="../../workspaces/applications/platform-applications/platform-application-requirements">Workspace platform application requirements</a>> to ensure that your cluster has sufficient resources.</p>
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/index.md:13]:
```suggestion
<p class="message--note"><strong>NOTE:</strong> Review the <a href="../mgmt-cluster-apps">Management cluster application requirements</a> and <a href="../../workspaces/applications/platform-applications/platform-application-requirements">Workspace platform application requirements</a>> to ensure that your cluster has sufficient resources.</p>
```

File: pages/dkp/kommander/2.2/install/air-gapped/index.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ Before installing, ensure you have:
 
 -   [MetalLB enabled and configured][air-gap-install-metallb], which provides load-balancing services.
 
+-   Sufficient resources on your cluster to run Kommander. Review [Management cluster application requirements](../mgmt-cluster-apps) and [Workspace platform application requirements](../../workspaces/applications/platform-applications/platform-application-requirements) for application requirements.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/air-gapped/index.md:36]:
```suggestion
-   Sufficient resources on your cluster to run Kommander. Review the [Management cluster application requirements](../mgmt-cluster-apps) and [Workspace platform application requirements](../../workspaces/applications/platform-applications/platform-application-requirements) for application requirements.
```

**Review by gvasquezvargas — APPROVED** [review]

**Review by msdolbey — APPROVED** [review]
👍 
Thanks!
Magnificent

**Review by gflorinskaya — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Pull request merged**